Extension/package.json

```diff
@@ -38,7 +38,25 @@
     "Linters"
   ],
```

Comment:
I think you need to undo this file change. I dont' think we should be changing our activation from *.